tools/visinity/macros.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<macros>
+    <xml name="requirements">
+        <requirements>
+            <container type="docker">quay.io/sargentl/vis_test:0.0.3</container>
```

Comment:
Should this still be named "test"?

Reply:
nope! this PR isn't quite ready for merger, i still have cleanup to do. this is a fork of mine as visinity had a pretty serious show-stopping bug when i started developing this IT; they've since updated it, so my plan is to ensure the new update works then use that container